chimney/src/test/scala/io/scalaland/chimney/PartialTransformerResultSpec.scala

```diff
@@ -0,0 +1,54 @@
+package io.scalaland.chimney
+
+import io.scalaland.chimney.dsl._
+import io.scalaland.chimney.examples.trip._
+import utest._
+
+object PartialTransformerResultSpec extends TestSuite {
```

Comment:
I would expect tests of just `Result` class here (map/flatMap/product/sequence/traverse x short/long circuit), not the PartialTransformer dsl usage...

Maybe also rename to `PartialResultSpec`?

Reply:
Renamed, tests are yes to write but tests that used to be there were moved to other suites or got removed if redundant.